index.js

```diff
@@ -71,6 +71,12 @@ module.exports = function (options) {
         parent.use(function startup(req, res, next) {
             if (promise.isPending()) {
                 res.status(503);
+                var headers = options.startupHeaders;
+                if(headers){
+                    Object.keys(headers).forEach(function(key){
+                        res.header(key, headers[key]);
+                    });
```

Comment:
`res.header` can also just [take an object](https://github.com/strongloop/express/blob/dee9fbbbda2f2483c657cf912f68db1a5ba1fcac/lib/response.js#L702-L704). No need to iterate.